src/js/components/AppListFilterComponent.jsx

```diff
@@ -71,13 +76,19 @@ var AppListFilterComponent = React.createClass({
   },
 
   blurInputGroup: function () {
-    console.log(this.state);
     this.setState({
       focused: false,
       activated: this.state.filterText !== ""
     });
   },
 
+  handleKeyDown: function (event) {
+    if (event.key === "Escape") {
```

Comment:
Yep, [that's right](https://facebook.github.io/react/docs/events.html#keyboard-events) - I was pretty happy to be able to use `key` instead of the usual `which`/`keyCode` dance.